examples/with-apollo-and-redux/lib/initClient.js

```diff
@@ -1,8 +1,6 @@
 import { ApolloClient, createNetworkInterface } from 'react-apollo'
 
-let apolloClient = null
-
-function _initClient (headers, initialState) {
+export const initClient = (headers, initialState = {}) => {
```

Comment:
This seems incorrect, you're creating a `new ApolloClient()` every time even on client side. On client side, you should store in the `apolloClient` var and just return that if it exists.

It looks like the code should just be left as it is.